CareKit/Connect/OCKConnectViewController.m

```diff
@@ -219,9 +219,11 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     OCKContact *contact = _sectionedContacts[indexPath.section][indexPath.row];
     
     OCKConnectDetailViewController *detailViewController = [[OCKConnectDetailViewController alloc] initWithContact:contact];
+    detailViewController.view.tintColor = self.view.tintColor;
     detailViewController.delegate = self.delegate;
     detailViewController.masterViewController = self;
     detailViewController.showEdgeIndicator = _showEdgeIndicators;
+    self.navigationController.navigationBar.tintColor = self.view.tintColor;
```

Comment:
Yep, I think that we should do that. If I remember well they do the same in RK.
Let's figure out how to approach the progress and the 'll provide a new PR